apps/examples/simple-example/api/openapi.json

```diff
@@ -1456,6 +1585,595 @@
+            "default": {
+              "docs_path": null
+            }
+          },
+          "engine_version": {
+            "type": "string",
+            "default": "0.6.0"
+          }
+        },
+        "x-module-name": "hopeit.server.config",
+        "description": "\n    Server configuration\n    "
+      },
+      "StreamsConfig": {
+        "type": "object",
+        "properties": {
+          "stream_manager": {
+            "type": "string",
+            "default": "hopeit.streams.NoStreamManager"
+          },
+          "connection_str": {
+            "type": "string",
+            "default": "<<NoStreamManager>>"
+          },
+          "delay_auto_start_seconds": {
+            "type": "integer",
+            "default": 3
+          }
+        },
+        "x-module-name": "hopeit.server.config",
+        "description": "\n    :field connection_str: str, url to connect to streams server: i.e. redis://localhost:6379\n    "
+      },
+      "LoggingConfig": {
+        "type": "object",
+        "properties": {
+          "log_level": {
+            "type": "string",
+            "default": "INFO"
+          },
+          "log_path": {
+            "type": "string",
+            "default": "logs/"
+          }
+        },
+        "x-module-name": "hopeit.server.config",
+        "description": "LoggingConfig(log_level: str = 'INFO', log_path: str = 'logs/')"
+      },
+      "AuthConfig": {
+        "type": "object",
+        "required": [
+          "secrets_location",
+          "auth_passphrase"
+        ],
+        "properties": {
+          "secrets_location": {
+            "type": "string"
+          },
+          "auth_passphrase": {
+            "type": "string"
+          },
+          "enabled": {
+            "type": "boolean",
+            "default": true
+          },
+          "create_keys": {
+            "type": "boolean",
+            "default": false
+          },
+          "domain": {
+            "type": "string"
+          },
+          "encryption_algorithm": {
+            "type": "string",
+            "default": "RS256"
+          },
+          "default_auth_methods": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "enum": [
+                "Unsecured",
+                "Basic",
+                "Bearer",
+                "Refresh"
+              ],
+              "x-enum-name": "AuthType",
+              "x-module-name": "hopeit.server.config"
+            },
+            "default": []
+          }
+        },
+        "x-module-name": "hopeit.server.config",
+        "description": "\n    Server configuration to handle authorization tokens\n    "
+      },
+      "APIConfig": {
+        "type": "object",
+        "properties": {
+          "docs_path": {
+            "type": "string"
+          }
+        },
+        "x-module-name": "hopeit.server.config",
+        "description": "APIConfig(docs_path: Union[str, NoneType] = None)"
```

Comment:
this line should be:
 "description": "APIConfig(docs_path: Optional[str] = None)"

Reply:
Optional[str] is actually an alias for Union[str, NoneType]